test/router-view.spec.js

```diff
@@ -1,10 +1,12 @@
 import {StageComponent} from 'aurelia-testing';
-import {RouteLoader, AppRouter, Router} from 'aurelia-router';
+import {RouteLoader, AppRouter, Router, RouteConfig, RouterConfiguration} from 'aurelia-router';
 import {TemplatingRouteLoader} from 'src/route-loader';
 import {testConstants} from 'test/test-constants';
 import {bootstrap} from 'aurelia-bootstrapper';
+import { inlineView } from 'aurelia-templating';
 
 describe('router-view', () => {
+  /**@type {StageComponent} */
```

Comment:
I'm not 100% sure of this syntax, is this adding types to JavaScript with TS through comments?

Reply:
yes, any IDE that understand TypeScript and jsDoc will give us intellisense for JS code, so I annotate those params. Tried to make it read easier